### ISSUE TITLE
rule: inline-no-min-max-dimensions

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#120]
## Rule ID
`inline-no-min-max-dimensions`

## What does this rule detect?
`min-width`, `max-width`, `min-height`, `max-height` on non-replaced inline elements have no effect, same as `width`/`height` (already covered by `inline-no-dimensions`).

## Example: should warn
```html
<span style="display: inline; max-width: 200px;">text</span>
```

## Example: should NOT warn
```html
<span style="display: inline-block; max-width: 200px;">text</span>
```

## Related CSS Properties
`display`, `min-width`, `max-width`, `min-height`, `max-height`

## References
- [CSS 2.2 §10.4 - min/max width](https://www.w3.org/TR/CSS22/visudet.html#min-max-widths): "do not apply" to non-replaced inline elements
- [Firefox inactive-property-helper.js](https://searchfox.org/mozilla-central/source/devtools/server/actors/utils/inactive-property-helper.js) — detects this pattern
- Natural complement to existing `inline-no-dimensions`
